docs/api.md

```diff
@@ -508,6 +508,17 @@ This example logs a message for a single page `load` event:
 page.once('load', () => console.log('Page loaded!'));
 ```
 
+To unsubscribe from events use the `removeListener` method:
+
+```js
+function onLoad() {
+  console.log('Page loaded!');
+}
+page.on('load', onLoad);
```

Comment:
This is fine, but it might be more clear if we use an invent that fires more than once. `page.on('console'...`?